src/compiler/checker.ts

```diff
@@ -11378,6 +11380,33 @@ module ts {
             return undefined;
         }
 
+        function getSymbolFromDeclarationKeyword(node: Node): Symbol {
+            switch(node.kind) {
+                case SyntaxKind.ClassKeyword:

```

Comment:
why enable all these keywords? We used to that and found it to be noisy, specially that QuickInfo is triggered whenever the mouse moves. function expressions is a different story, they do not have a name, so there is no target to land on any ways. that is not the case for all other declarations, they all have names.

i would not enable all these, and only special case `function`.